code-challenges/string.md

```diff
@@ -176,3 +327,20 @@ Given an "out" string length 4, such as "<<>>", and a word, return a new string
 make_out_word('<<>>', 'Yay') → '<<Yay>>'
```

Comment:
@Yosolita1978, for this question, is `out` always going to be of length 4? or should this solution also handle something like `##[]##`?

Reply:
For this problem, you can assume that `out` will always be length 4.